.github/workflows/ci/helm.yaml

```diff
@@ -0,0 +1,19 @@
+name: ci/helm
+
+on:
+  pull_request:
+    paths:
+      - 'helm/**'
+  push:
+    branches:
+      - main
+    paths:
+      - 'helm/**'
```

Comment:
This means it only runs when the helm chart actually changes? Nice.

Reply:
That's right. One issue with the CI workflow previously is it unconditionally runs CI for all components regardless if there are changes to those components.